ubuntu/ubuntu-20.x/ubuntu-20.04-hpc/install_prerequisites.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -ex
+
+# jq is needed to parse the component versions from the requirements.json file
+apt-get update
+apt-get install -y jq
```

Comment:
Add a line space at the end of this file

Reply:
I've updated set-properties.sh and install_prerequisites.sh adding blank line at end of script.